src/pages/workspaces/workspace/Data.js

```diff
@@ -177,7 +178,11 @@ class WorkspaceDataContent extends Component {
             _.toPairs(entityMetadata))),
           div({ style: { display: 'flex', alignItems: 'center', justifyContent: 'space-between', ...styles.dataTypeHeading } }, [
             'Reference Data',
-            link({ onClick: () => this.setState({ importingReference: true }) }, [icon('plus-circle')])
+            link({
```

Comment:
It seems like what we want here is a `Clickable` that shares the link styling. Could we pull that out and share it, instead of changing `link`?